app/services/credit_notes/create_service.rb

```diff
@@ -175,6 +176,12 @@ def should_handle_refund?
       invoice_payment.present?
     end
 
+    def should_report_to_tax_provider?
```

Comment:
This check is unnecessary here. This snippet is wrapped under `credit_note.finalized?` and check `customer.anrok_customer` is present inside the job so that we avoid this query in the service

Reply:
👍